src/polling.ts

```diff
@@ -52,6 +53,13 @@ export class PollingProcessor {
       this.timeout = setTimeout(() => this.poll(), sleepFor);
     };
 
+    if (this.lastPollTime > (Date.now() - this.options.pollInterval)) {
+      this.log.info(`Last poll was ${(Date.now()-this.lastPollTime)/1000} seconds ago, skipping flag refresh`);
```

Comment:
This will almost certainly wind up being a non-whole number. We're best passing it through either `Math.round` or `Math.ceil` to make it more readable.

```suggestion
      this.log.info(`Last poll was ${(Date.now()-this.lastPollTime)/1000} seconds ago, skipping flag refresh`);
```